lib/Graph/Graph.cpp

```diff
@@ -1043,8 +1043,13 @@ CmpEQNode *Function::createCmpEQ(llvm::StringRef name, NodeValue LHS,
   return addNode(new CmpEQNode(name, OT, LHS, RHS));
 }
 
+PowNode *Function::createPow(llvm::StringRef name, NodeValue Base, NodeValue Exp) {
```

Comment:
nit: While we're refactoring, maybe change this to `base` and `exp`? Here and in the other `createPow()` too.

Reply:
Done. But I'm not sure what is out convention here. I see both `RHS` and `rhs`, also `Cond` and `Ty` and `ty`

Reply:
Yeah it's a bit inconsistent in some places. I think `RHS` makes sense over `rhs` since it's an acronym. I think all others should be camel case, e.g. `cond` and `ty`.